nodes/test/test_chroma_filter_semantics.py

```
@@ -0,0 +1,203 @@
+# =============================================================================
+
+"""Regression tests for Chroma filter conversion semantics."""
+
+from __future__ import annotations
+
+import importlib.util
+import sys
+import types
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Iterator
```

Ruff (0.15.6)

[warning] Import from `collections.abc` instead: `Iterator`

Import from `collections.abc`

(UP035)